src/components/common/Slider.ts

```diff
@@ -0,0 +1,28 @@
+import 'rc-slider/assets/index.css';
+
+import Slider from 'rc-slider';
+import { h } from 'react-hyperscript-helpers';
+import colors from 'src/libs/colors';
+
+const handleStyle = {
+  borderColor: colors.accent(),
+  backgroundColor: colors.accent(),
+  opacity: 1,
+  boxShadow: 'none',
+  height: 18,
+  width: 18,
+  zIndex: 0,
+};
+const trackHeight = 8;
+const trackStyle = { backgroundColor: colors.accent(0.5), height: trackHeight };
+const railStyle = { backgroundColor: colors.accent(0.4), height: trackHeight };
+
+const TerraSlider = (props) =>
+  h(Slider, {
+    handleStyle,
```

Comment:
I figure we probably want consistent styling across the app, so I pulled the styles in here.